multimodal/tests/unittests/others/test_semantic_segmentation.py

```diff
@@ -0,0 +1,107 @@
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+import torch
+
+from autogluon.core.utils.loaders import load_zip
+from autogluon.multimodal import MultiModalPredictor
+
+
+def get_file_paths(directory):
+    file_paths = sorted(os.listdir(directory))
+    return [os.path.join(directory, file_path) for file_path in file_paths]
+
+
+def download_sample_dataset():
+    zip_file = "https://automl-mm-bench.s3.amazonaws.com/unit-tests/tiny_isic2017.zip"
+    download_dir = "./tiny_isic2017"
+
+    load_zip.unzip(zip_file, unzip_dir=download_dir)
+    data_dir = os.path.join(download_dir, "tiny_isic2017")
+
+    return data_dir
+
+
+@pytest.mark.parametrize(
+    "checkpoint_name",
+    [
+        "facebook/sam-vit-base",
+    ],
+)
+def test_sam_semantic_segmentation_fit_evaluate_predict_isic(checkpoint_name):
+    data_dir = download_sample_dataset()
+    train_img_files = get_file_paths(os.path.join(data_dir, "train/ISIC-2017_Train"))
+    train_gt_files = get_file_paths(os.path.join(data_dir, "train/ISIC-2017_Training_Part1_GroundTruth"))
+    val_img_files = get_file_paths(os.path.join(data_dir, "val/ISIC-2017_Val"))
+    val_gt_files = get_file_paths(os.path.join(data_dir, "val/ISIC-2017_Validation_Part1_GroundTruth"))
+    test_img_files = get_file_paths(os.path.join(data_dir, "test/ISIC-2017_Test"))
+    test_gt_files = get_file_paths(os.path.join(data_dir, "test/ISIC-2017_Test_v2_Part1_GroundTruth"))
+
+    train_df = pd.DataFrame({"image": train_img_files, "label": train_gt_files})
+    val_df = pd.DataFrame({"image": val_img_files, "label": val_gt_files})
+    test_df = pd.DataFrame({"image": test_img_files, "label": test_gt_files})
```

Comment:
This block looks like duplicated in the two tests. Consider wrapping it into a util function.